packages/rdma-core/rdma-core.spec

```diff
@@ -0,0 +1,157 @@
+%global abiver rdmav34
+
+Name: %{_cross_os}rdma-core
+Version: 54.0
+Release: 1%{?dist}
+Summary: RDMA core userspace infrastructure, including core libraries and util programs.
+License: Linux-OpenIB AND MIT
+Source0: https://github.com/linux-rdma/rdma-core/releases/download/v%{version}/rdma-core-%{version}.tar.gz
+Source100: libibverbs-tmpfiles.conf
+
+# RDMA logdog configuration
+Source200: logdog.rdma.conf
+
+BuildRequires: cmake
+BuildRequires: %{_cross_os}libnl-devel
+BuildRequires: %{_cross_os}glibc-devel
+Requires: %{_cross_os}libnl
+
+%description
+%{summary}.
+
+%package devel
+Summary: RDMA core development libraries and headers
+%description devel
+%{summary}.
+
+%prep
+%autosetup -n rdma-core-%{version} -p1
+
+%build
+%{cross_cmake} . \
+  -DNO_PYVERBS=1 \
+  -DNO_MAN_PAGES=1 \
+  -DCMAKE_BUILD_TYPE=Release \
```

Comment:
nit: you don't need to set `CMAKE_BUILD_TYPE=Release`, it is already set by `cross_cmake`